tools/benchmark/cmd/put.go

```diff
@@ -52,6 +56,9 @@ func init() {
 	putCmd.Flags().IntVar(&putTotal, "total", 10000, "Total number of put requests")
 	putCmd.Flags().IntVar(&keySpaceSize, "key-space-size", 1, "Maximum possible keys")
 	putCmd.Flags().BoolVar(&seqKeys, "sequential-keys", false, "Use sequential keys")
+	putCmd.Flags().BoolVar(&sameKey, "same-key", false, "Use one same key (useful for compaction benchmark)")
```

Comment:
we do not need same key option. if you specify key-space-size == 1, it will always put on the same key.